src/dao/databaseDAO.py

```diff
@@ -0,0 +1,20 @@
+from environment.database_objects.config import *
+from sqlalchemy import text
+from sqlalchemy.orm import declarative_base, sessionmaker
+from sqlalchemy.sql import func
+from sqlalchemy import create_engine
+
+class DBDAO:
```

Comment:
@sweiyang , This way the DBDAO can be reused across all the DAO objects by class UserDAO(DBDAO): or 
class TaskDAO(DBDAO):